sdks/python/apache_beam/ml/inference/pytorch_inference_test.py

```diff
@@ -63,6 +63,17 @@
              for f1, f2 in TWO_FEATURES_EXAMPLES]).reshape(-1, 1))
 ]
 
+TWO_FEATURES_DICT_OUT_PREDICTIONS = [
+    PredictionResult(ex, pred) for ex,
+    pred in zip(
+        TWO_FEATURES_EXAMPLES,
+        [{
+            "output1": torch.Tensor([f1 * 2.0 + f2 * 3 + 0.5]),
+            "output2": torch.Tensor([f1 * 2.0 + f2 * 3 + 0.5])
+        } for f1,
+         f2 in TWO_FEATURES_EXAMPLES])
+]
```

Comment:
Could we somehow use the existing `TWO_FEATURES_PREDICTIONS` to construct `TWO_FEATURES_DICT_OUT_PREDICTIONS` so that if the former changes the computation, the latter will follow?

Reply:
That's probably fine - I was aiming for greater completeness/simplicty modularity/DRY (see subheader [Make Your Tests Complete and Concise](https://abseil.io/resources/swe-book/html/ch12.html) for a more fleshed out philosophy),  but I agree this ends up being better. Done